test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,20 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    def get_metadata_max_size(self, cache_line_size, number_of_cores=4096):
```

Comment:
Maybe make this method static and instead of _self_ set _cache_size_ as the first parameter, so user would not have to have running cache to count metadata max size